cirq_superstaq/custom_gates.py

```diff
@@ -572,6 +572,28 @@ def _json_dict_(self) -> Dict[str, Any]:
         return cirq.protocols.obj_to_dict_helper(self, ["theta", "phi"])
 
 
+class StrippedCZGate(cirq.Gate):
+    def __init__(self, phi=0) -> None:
+        super(StrippedCZGate, self)
+        self.phi = phi
+
+    def _num_qubits_(self) -> int:
+        return 2
+
+    def _unitary_(self) -> Optional[npt.NDArray[np.complex_]]:
+        return np.array(
+            [
+                [1.0, 0.0, 0.0, 0.0],
+                [0.0, 1.0, 0.0, 0.0],
+                [0.0, 0.0, 1.0, 0.0],
```

Comment:
I think these should be phases too
```suggestion
                [0.0, np.exp(1j * self.phi), 0.0, 0.0],
                [0.0, 0.0, np.exp(1j * self.phi), 0.0],
```

Reply:
Ah right good catch.